synapse/rest/media/v1/preview_html.py

```diff
@@ -281,7 +281,7 @@ def parse_html_description(tree: "etree.Element") -> Optional[str]:
 
 
 def _iterate_over_text(
-    tree: "etree.Element", *tags_to_ignore: Iterable[Union[str, "etree.Comment"]]
+    tree: "etree.Element", *tags_to_ignore: Union[str, "etree.Comment"]
```

Comment:
Honestly, I think it doesn't even make sense to pass this, we should probably just make `TAGS_TO_REMOVE` a constant.

Reply:
(Or at the very list, not splat it?)

Reply:
Yeah, pretty much! 👍